views/support/history.tt.text

```diff
@@ -0,0 +1,49 @@
+;; -*- coding: utf-8 -*-
+
+.title=Support Request History
+
+.error.invalidemail=Invalid email to search on.
+
+.error.invaliduser=Invalid user to search on.
+
+.error.nodatabase=Failed to get database handle.
+
+.noresults=No results found for the search terms you entered.
+
+.return=Back to account results
+
+.search.btn=Perform Search
+
+.search.text=Search for Requests
+
+.search.email=By Email:
+
+.search.email.text=Email Address
+
+.search.user=By User:
+
+.search.user.text=Username
+
+.search.userid=By Userid:
+
+.search.userid.text=Userid
+
+.search.onefield=Please fill out only one of the search fields.
+
+.search.otheremails=You may also search for requests from email addresses associated with your account.
+
+.search.otheremails.label=Search
+
+.table.category=Category
+
+.table.summary=Summary
+
+.table.state=State
+
+.table.answeredby=Answered By
+
+.table.openedby=Opened By
+
+.table.timeopened=Time Opened
+
+.viewing=Viewing support requests for [[username]]
```

Comment:
Your history.tt.text file needs to have its keys in alphabetical order, for easy reference.